inc/classes/admin/settings/class-settings.php

```diff
@@ -367,6 +367,13 @@ public function sanitize_callback( $input ) {
 			$input['cloudflare_api_key'] = WP_ROCKET_CF_API_KEY;
 		}
 
+		// Check Cloudflare input data and display error message
```

Comment:
You should check if the main CF option `do_cloudflare` is enabled before testing and sending back any error, else it will show an error for something the user is not using.

Reply:
Done. I had Cloudflare enabled and forgot to test it without do_cloudflare. Thank you for spotting this issue!